src/main/java/com/google/cloud/tools/appengine/api/whitelist/AppEngineJreWhitelist.java

```diff
@@ -0,0 +1,1979 @@
+              "javax.lang.model.element.Parameterizable",
+              "javax.lang.model.element.QualifiedNameable",
+              "javax.lang.model.element.TypeElement",
+              "javax.lang.model.element.TypeParameterElement",
+              "javax.lang.model.element.UnknownAnnotationValueException",
+              "javax.lang.model.element.UnknownElementException",
+              "javax.lang.model.element.VariableElement",
+              "javax.lang.model.type.ArrayType",
+              "javax.lang.model.type.DeclaredType",
+              "javax.lang.model.type.ErrorType",
+              "javax.lang.model.type.ExecutableType",
+              "javax.lang.model.type.MirroredTypeException",
+              "javax.lang.model.type.MirroredTypesException",
+              "javax.lang.model.type.NoType",
+              "javax.lang.model.type.NullType",
+              "javax.lang.model.type.PrimitiveType",
+              "javax.lang.model.type.ReferenceType",
+              "javax.lang.model.type.TypeKind",
+              "javax.lang.model.type.TypeMirror",
+              "javax.lang.model.type.TypeVariable",
+              "javax.lang.model.type.TypeVisitor",
+              "javax.lang.model.type.UnknownTypeException",
+              "javax.lang.model.type.WildcardType",
+              "javax.lang.model.util.AbstractAnnotationValueVisitor6",
+              "javax.lang.model.util.AbstractElementVisitor6",
+              "javax.lang.model.util.AbstractTypeVisitor6",
+              "javax.lang.model.util.ElementFilter",
+              "javax.lang.model.util.ElementKindVisitor6$1",
+              "javax.lang.model.util.ElementKindVisitor6",
+              "javax.lang.model.util.ElementScanner6",
+              "javax.lang.model.util.Elements",
+              "javax.lang.model.util.SimpleAnnotationValueVisitor6",
+              "javax.lang.model.util.SimpleElementVisitor6",
+              "javax.lang.model.util.SimpleTypeVisitor6",
+              "javax.lang.model.util.TypeKindVisitor6",
+              "javax.lang.model.util.Types",
+              "javax.script.AbstractScriptEngine",
+              "javax.script.Bindings",
+              "javax.script.Compilable",
+              "javax.script.CompiledScript",
+              "javax.script.Invocable",
+              "javax.script.ScriptContext",
+              "javax.script.ScriptEngine",
+              "javax.script.ScriptEngineFactory",
+              "javax.script.ScriptEngineManager",
+              "javax.script.ScriptException",
+              "javax.script.SimpleBindings",
+              "javax.script.SimpleScriptContext",
+              "javax.sql.CommonDataSource",
+              "javax.sql.ConnectionEvent",
+              "javax.sql.ConnectionEventListener",
+              "javax.sql.ConnectionPoolDataSource",
+              "javax.sql.DataSource",
+              "javax.sql.PooledConnection",
+              "javax.sql.RowSet",
+              "javax.sql.RowSetEvent",
+ 
```

Comment:
I'm confused what's going on here. Are these not covered by the whitelist itself?